### DISCUSSION TITLE
关于使用unicode输入特殊字符的模块(已解决)

### DISCUSSION THREAD

**Discussion by algebnaly** [fcitx/fcitx5#618] [General]
通过快捷键ctrl+shift+alt+U 唤起的unicode 输入模块只能通过鼠标选择将要输入的字符吗?
如果是这样我会尝试提交PR来添加通过**ctrl+shift+alt+空格键**输入第一个候选的字符的特性。
(我的键盘上等于号和字母O键不灵敏,时常需要unicode输入模块来输入这些字符, 如果能够像拼音输入那样之间通过一个快捷键输入第一个候选的字符就太好了。)

**Comment by wengxt** [comment]
A-xx 表示 Alt +  数字键选择

**Reply by algebnaly** [reply]
明白了，谢谢！
